### PR TITLE
Add a landing page

### LINKED ISSUE #7: Build out scucess page
Page users see after submitting the form. 

- [ ] tells them it worked
- [ ] lists who they sent their message to in case they forgot
- [ ] has link to share on facebook
- [ ] has link to toot it in on twitter
- [ ] looks good on mobile and desktop

### PR DESCRIPTION
**Pull request by dbalatero**
screenshot 2016-03-30 16 18 55

Closes #7.

### CODE REVIEW

File: app/controllers/landing_controller.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class LandingController < ApplicationController
+  def index
+    @states = States.supported
```
Comment (bion) [inline app/controllers/landing_controller.rb]: Why not `Delegate.pluck(:state).uniq`?

**Comment by bion** [comment]
@dbalatero 1 comment thn :ship:

File: app/controllers/landing_controller.rb (hunk 1 above)
Comment (dbalatero) [inline app/controllers/landing_controller.rb]: I didn't have access to that code at the time I wrote it?

Comment (dbalatero) [inline app/controllers/landing_controller.rb]: I only had the `Message` model available to me at the time…

File: app/models/states.rb [hunk 2]
```diff
@@ -0,0 +1,67 @@
+class States
+  def self.supported
+    state_codes = Set.new(Delegate.pluck("distinct state"))
+
+    STATES.reject do |code, name|
+      !state_codes.include?(code)
+    end
```
Comment (bion) [inline app/models/states.rb]:
```
STATES.select do |code, name|
  state_codes.include?(code)
end 
```

**Pull request merged**